lib/util/util.js

```diff
@@ -764,6 +764,26 @@ const Util = {
         const options = { stdio: [0, 1, 2] };
         return child_process.execSync(cmd + ' ' + args.join(' '), options);
     },
+    /**
+     * helper to standardize retrieveTemplate result checks
```

Comment:
Unclear what this does

Reply:
basically idea is to check that exactly 1 item is returned, then use it in other RT methods.